client/my-sites/site-settings/form-general.jsx

```diff
@@ -133,11 +135,23 @@ class SiteSettingsFormGeneral extends Component {
 
 		if (
 			this.props.isSavingSettings &&
-			! nextProps.isSavingSettings &&
-			nextProps.isSaveRequestSuccessful
+			! nextProps.isSavingSettings
 		) {
-			nextProps.clearDirtyFields();
-			nextProps.markSaved();
+			if ( nextProps.isSaveRequestSuccessful ) {
+				nextProps.successNotice( nextProps.translate( 'Settings saved!' ), { id: 'site-settings-save' } );
+				nextProps.clearDirtyFields();
+				nextProps.markSaved();
+			} else {
+				let text;
+				switch ( nextProps.saveRequestError.error ) {
+					case 'invalid_ip':
+						text = nextProps.translate( 'One of your IP Addresses was invalid. Please try again.' );
```

Comment:
I can't seem to trigger this error in the DNS manager.  There is some inline validation that requires xxx.xxx.xxx.xxx format, entering in a bogus IP seems to persist like 255.255.255.0 - anyhow, thinking it is fine to leave as-is